### ISSUE TITLE
TagNode - filter by global tags, exclude

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#357]
A lot of people are using global tags + a venn to "exclude artifacts" - if we added the ability to run global tags with an input, and have exclude, we could simplify things a bit

**Comment by davmlaw** [comment]
One this has passed let WP know so she can alter somatic analysis templates

**Comment by sksmi** [comment]
Took me a bit to understand the available options here.  Could we do something like the following? Minor suggestions, but think it might help. 

![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/077467f5-51c7-4ba9-b79e-f11c792d796a)

Not sure I'm completely understanding the exclude option. Is it expected that we can apply exclude on global tags, i.e. when the parent isn't selected? Testing global tags + exclude returns more variants than the all variants node (e.g. image below) - is it because the tags aren't build-specific? I'm not sure this feature is necessary, might be easier to hide exlcude option unless parent selected?
![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/7fb1ed80-61e2-41cf-8c9a-88717a95caa9)

**Comment by davmlaw** [comment]
Moved exclude to the bottom, added note.

Made exclude only work when parent input is selected.

I originally left that in because I figured why stop someone if they wanted to see all the variants that aren't tagged, but yeah it's confusing, I'd need to put a note about it being out of date as variants are added etc, and in the rare cases they want to do that, they can make it beneath the all variants node.
